create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/test/utils/ScopusGenerator.java

```diff
@@ -162,12 +176,27 @@ private static HeadTp randomHeadTp() {
         return head;
     }
 
-    private static CitationInfoTp randomCitationInfo() {
+    private CitationInfoTp randomCitationInfo() {
         var citationInfo = new CitationInfoTp();
         citationInfo.setAuthorKeywords(randomAuthorKeywordsTp());
+        citationInfo.getCitationType().add(createCitationType());
         return citationInfo;
     }
 
+    private CitationTypeTp createCitationType() {
+        var citationType = new CitationTypeTp();
+        citationType.setCode(createCitationTypeAtt());
+        return citationType;
+    }
+
+    private CitationtypeAtt createCitationTypeAtt() {
+        return nonNull(citationtypeAtt) ? citationtypeAtt : randomSupportedCitationType();
+    }
+
+    private CitationtypeAtt randomSupportedCitationType() {
```

Comment:
Add a TODO here to enrich the method when we support more citation types

Reply:
Fixed

Reply:
Fixed